src/app/builders/BuildersPage.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import { useAccount } from 'wagmi'
+import { ActionsContainer } from './components/Container'
+import { ActionMetricsContainer } from './components/Container/ActionMetricsContainer'
+import { InfoContainer } from './components/Container/InfoContainer'
+import { MetricsContainer } from './components/Container/MetricsContainer'
+import { PageTitleContainer } from './components/Container/PageTitleContainer'
+import { Metrics } from './components/Metrics'
+import { Content } from './components/Content'
+import { Spotlight } from './components/Spotlight'
+import { Table } from './components/Table'
+
+const NAME = 'Builders'
+
+export const BuildersPage = () => {
+  const { address } = useAccount()
+
+  const TableTitle = () => (
+    <div className="flex items-center justify-between w-full">
+      <h3 className="text-lg font-semibold text-white">THE COLLECTIVE BUILDERS</h3>
```

Comment:
I think that we have a component for `Typography`.  you can find an example of ths usage [here](https://github.com/RootstockCollective/dao-frontend/blob/main/src/app/backing/components/Container/PageTitleContainer.tsx#L13).

I also will avoid using UPPER CASES and use a styling :)